Assets/Scripts/GameManager.cs

```diff
@@ -45,6 +48,10 @@ void Update()
     {
 
     }
+    public string PlayerPrefsCharacterUnlock(int characterNum)
```

Comment:
```suggestion
    public string PlayerPrefsCharacterUnlockKey(Character _character)
```
캐릭터 enum을 명시적으로 입력받아야, 어떤 조건을 달성했을때 어떤 캐릭터가 해금되는지 명시적으로 파악할 수 있어서 더 좋습니다~